dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl

```diff
@@ -14,7 +14,7 @@ BEGIN_PROLOG
 dune10kt_1x2x2_gammas_in_APAboards:  # 3.81 cm thick (2*4*4.76 mm), 16.5 cm in the vertical direction, and as wide as the APA frames on top of upper APAs and bottom of lower APAs - J. Reichenbacher (12/11/2023)
 {
    module_type:           "RadioGen"
-   PathToFile:             "Flux/Radionuclides/v03_06/"
+   PathToFile:             "Flux/Radionuclides/v03_05/"
```

Comment:
Just double checking, is this downgrade here correct?

Reply:
We may want to change it in the future but turns out that when the flux file is the same as a previous version, they are not being copied to the new version folder, we are using what is already there.

Reply:
Got it, thanks for clarifying